ccpp/physics_namelists/input_GFS_v17_HR3_RRTMGP.nml

```diff
@@ -0,0 +1,171 @@
+&gfs_physics_nml
+  fhzero       = 6
+  h2o_phys     = .true.
+  ldiag3d      = .true.
+  qdiag3d      = .true.
+  print_diff_pgr = .false.
+  fhcyc        = 24
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = 8
+  iovr         = 3
+  ltaerosol    = .false.
+  lradar       = .true.
+  ttendlim     = -999
+  dt_inner     = 225.0
+  sedi_semi    = .true.
+  decfl        = 10
+  oz_phys      = .false.
+  oz_phys_2015 = .true.
+  lsoil_lsm    = 4
+  do_mynnedmf  = .false.
+  do_mynnsfclay = .false.
+  icloud_bl    = 1
+  bl_mynn_edmf = 1
+  bl_mynn_tkeadvect = .true.
+  bl_mynn_edmf_mom = 1
+  do_ugwp      = .false.
+  do_tofd      = .false.
+  gwd_opt      = 2
+  do_ugwp_v0   = .false.
+  do_ugwp_v1   = .true.
+  do_ugwp_v0_orog_only = .false.
+  do_ugwp_v0_nst_only  = .false.
+  do_gsl_drag_ls_bl    = .true.
+  do_gsl_drag_ss       = .true.
+  do_gsl_drag_tofd     = .true.
+  do_ugwp_v1_orog_only = .false.
+  min_lakeice  = 0.15
+  min_seaice   = 0.15
+  use_cice_alb = .false.
+  pdfcld       = .false.
+  fhswr        = 1200.
+  fhlwr        = 1200.
+  ialb         = 1
```

Comment:
Per the setting in RTs, it looks like ialb/iems=2 is more convincing for GFS suite with Noahmp.  We may need to double-check it.